src/Microsoft.AspNet.Server.Kestrel/App.config

```diff
@@ -0,0 +1,4 @@
+﻿<configuration>
+  <dllmap dll="libuv.dll" target="libuv.so.1" os="linux"/>
+  <dllmap dll="libuv.dll" target="native/darwin/universal/libuv.dylib" os="osx"/>
```

Comment:
Just checked with @GrabYourPitchforks and it sounds like you wouldn't expect a difference between DllImport P/Invoke and GetProcAddress delegate P/Invoke, as long as the same delegate instances are used repeatedly.


________________________________
From: Mark Rendle <notifications@github.com>
Sent: Monday, February 9, 2015 11:22 AM
To: aspnet/KestrelHttpServer
Cc: Louis DeJardin
Subject: Re: [KestrelHttpServer] P/Invoke cleanup (#60)


In src/Microsoft.AspNet.Server.Kestrel/App.config<https://github.com/aspnet/KestrelHttpServer/pull/60#discussion_r24356378>:

> @@ -0,0 +1,4 @@
> +?<configuration>
> +  <dllmap dll="libuv.dll" target="libuv.so.1" os="linux"/>
> +  <dllmap dll="libuv.dll" target="native/darwin/universal/libuv.dylib" os="osx"/>


Has anyone done relative performance testing on Kestrel before and after these changes? Happy to do so tomorrow on Linux and Windows, can't do Mac.

-
Reply to this email directly or view it on GitHub<https://github.com/aspnet/KestrelHttpServer/pull/60/files#r24356378>.